addons/survey/static/tests/components/question_page_one2many_field_tests.js

```diff
@@ -301,4 +304,60 @@ QUnit.module("QuestionPageOneToManyField", (hooks) => {
             assert.strictEqual(target.querySelector(".o_is_section [name=title]").innerText, "a");
         }
     );
+
+    QUnit.test("full page list question creation", async (assert) => {
+        const questions = [];
+        for (let i = 1; i <= 45; i++) {
+            questions.push({
+                id: i,
+                is_page: false,
+                title: `question ${i}`,
+                sequence: i,
+            });
+        }
+
+        serverData.models.survey_question.records = questions;
+        serverData.models.survey.records[0].question_and_page_ids = questions.map((q) => q.id);
+        await makeView({
+            type: "form",
+            resModel: "survey",
+            resId: 1,
+            serverData,
+            arch: `
+            <form>
+                <field name="question_and_page_ids" widget="question_page_one2many">
+                    <tree>
+                        <field name="is_page" invisible="1" />
+                        <field name="title" />
+                        <field name="random_questions_count" />
+                        <field name="sequence" widget="handle"/>
+                    </tree>
+                </field>
+            </form>
+        `,
+            mockRPC(route, args) {
+                if (args.method === "web_save" && args.model === "survey") {
+                    const sequence = args.args[1].question_and_page_ids[0][2].sequence;
+                    assert.step(`question sequence ${sequence}`);
+                }
```

Comment:
With the change, I think the sequence are not increased by one. So I would change the test like this:
```javascript
let lastQuestionSequence = 45;
// ...
mockRPC(route, args) {
    if (args.method === "web_save" && args.model === "survey") {
        const sequence = args.args[1].question_and_page_ids[0][2].sequence;
        assert.ok(sequence > lastQuestionSequence);
        assert.step("question sequence increasing");
        lastQuestionSequence = sequence;
    }
},
// ...
assert.verifySteps(["question sequence increasing"]);
```
What do you think ?